packages/in-percy/index.js

```diff
@@ -0,0 +1,6 @@
+// Returns true if executing in the Percy rendering environment
+export default function inPercy() {
```

Comment:
@fotinakis FWIW, there is a linter applied to this repo, just not one consistent with the styles we use elsewhere. 😆